zebrad/src/components/mempool/error.rs

```diff
@@ -9,22 +12,47 @@ use super::storage::{
     ExactTipRejectionError, SameEffectsChainRejectionError, SameEffectsTipRejectionError,
 };
 
+/// Mempool errors.
 #[derive(Error, Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(any(test, feature = "proptest-impl"), derive(Arbitrary))]
 pub enum MempoolError {
+    /// Transaction rejected based on its authorizing data (scripts, proofs,
+    /// signatures),
```

Comment:
I think we might have missed this part of the docs:

```suggestion
    /// signatures). The rejection is valid for the current chain tip.
```